flink-runtime/src/main/java/org/apache/flink/runtime/io/network/NetworkEnvironment.java

```diff
@@ -129,11 +128,11 @@ public static NetworkEnvironment fromConfiguration(
 			maxJvmHeapMemory,
 			localTaskManagerCommunication,
 			taskManagerAddress);
-		return NetworkEnvironment.create(networkConfig, taskEventPublisher, metricGroup, ioManager);
```

Comment:
This should be done in previous hotfix commit while introducing `fromConfiguration` firstly.